pkgs/development/python-modules/imagededup/default.nix

```
@@ -0,0 +1,44 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, cython
+, torch
+, torchvision
+, pillow
+, tqdm
+, scikit-learn
+, pywavelets
+, matplotlib
+}:
+
+buildPythonPackage rec {
+  pname = "imagededup";
+  version = "0.3.2";
```

```suggestion
  version = "0.3.2";
  format = "setuptools";
```
Please add `format` explicitly.